README.md

```diff
@@ -80,12 +80,12 @@ curl http://localhost:10443/transfer?queue=AUDIT
 
 ### Purge
 
-`GET /purgedlq?queue=<<QUEUE>>`
+`GET /purge?queue=<<QUEUE>>&dlq=<<BOOL>>`
 
-Deletes all messages on the dead letter queue. To be used when the message can never be processed successfully.
+Deletes all messages from the specified queue.
 
 ```sh
-curl http://localhost:10443/purgedlq?queue=AUDIT
```

Comment:
Not blocking, but what about no querystring and use AUDIT and AUDIT_DLQ, it's a bigger refactor though isn't it?

Reply:
I'd prefer to use query parameters being honest - it makes it clear what you are requesting. Because DLQs are linked to Queues it makes sense...